src/NetworkSettings.cpp

```diff
@@ -294,6 +296,17 @@ void NetworkSettingsClass::setStaticIp()
     }
 }
 
+void NetworkSettingsClass::setupMDNS()
+{
+    MessageOutput.print(F("Setting up mDNS responder... "));
+    if (!MDNS.begin(getHostname().c_str())) {
+        MessageOutput.print(F("failed"));
```

Comment:
I would also announce the OpenDTU service here.

```cpp
MDNS.addService("http", "tcp", 80);
MDNS.addService("opendtu", "tcp", 80); // So that devices can specifically search for OpenDTU instances
```

Reply:
We should advertise the username & password for initial login and direct new users to the password setup page as part of auto-discovery:
```c++
MDNS.addServiceTxt("http", "tcp", "u", "admin");
MDNS.addServiceTxt("http", "tcp", "p", "openDTU42");
MDNS.addServiceTxt("http", "tcp", "path", "/settings/network");
```
I would vote for changing the order in Settings > Network Settings to make the Hostname more prominent:
* Hostname
* WiFi SSID
* WiFi Password
* DHCP (yes/no)

We could also advertise the MQTT topic if that makes sense ?
```c++
MDNS.addService("mqtt", "tcp", "topic", "/inverter");
```

Reply:
Mqtt Topic seems bit unnecessary. Also, mDNS is not really intended for username & password.